content/200-concepts/200-database-connectors/05-sqlite.mdx

```diff
@@ -42,7 +42,14 @@ The SQLite connector maps the [scalar types](/concepts/components/prisma-schema/
 | `Float`    | `REAL`        |
 | `Decimal`  | `DECIMAL`     |
 | `Json`     | Not supported |
-| `Bytes`    | `BLOB`       |
+| `Bytes`    | `BLOB`        |
+
+## Rounding errors on big numbers
+
+SQLite is a loosely-typed database. If your Schema has a field with count `Int`, Prisma prevents you from inserting a value larger than an integer. However, nothing prevents the database from directly accepting a bigger number. These manually inserted big numbers cause rounding errors when queried. To avoid this problem, Prisma 4.0.0 and later checks numbers on the way out of the database to verify that they fit within the boundaries of an integer.
+
+**!!!What happens if they don't - what error does it throw? !!!**
```

Comment:
What I mean here is what happens if a big number doesn't fit within the boundaries of an integer